src/components/sbb-train-wagon/sbb-train-wagon.tsx

```diff
@@ -37,10 +36,10 @@ export class SbbTrainWagon extends LitElement {
   public static override styles: CSSResult = Style;
 
   /** Wagon type. */
-  @property({ reflect: true }) public type: InterfaceSbbTrainWagonAttributes['type'] = 'wagon';
+  @property({ reflect: true }) public type: 'locomotive' | 'closed' | 'wagon' = 'wagon';
 
   /** Occupancy of a wagon. */
-  @property() public occupancy?: InterfaceSbbTrainWagonAttributes['occupancy'] = 'unknown';
+  @property() public occupancy?: 'high' | 'medium' | 'low' | 'unknown' = 'unknown';
```

Comment:
Could be added global, as it will be used in the new occupancy component, and maybe already in the timetable-row?

Reply:
Created a new type, not sure about timetable-row, there's a different enum there. 
For sure it could be useful for the new occupancy component.